build/automation/var/profile/local.mk

```diff
@@ -4,6 +4,7 @@
 # Service variables
 
 API_HOST = api
+API_ADMIN_ALLOWED_HOSTS = 127.0.0.1
```

Comment:
Can we have a corresponding line in the `dev.mk` profile? I.e. `#API_ADMIN_ALLOWED_HOSTS = [secret]`